clustergroup/templates/applications.yaml

```diff
@@ -147,11 +147,13 @@ spec:
   {{- if .ignoreDifferences }}
   ignoreDifferences: {{ .ignoreDifferences | toPrettyJson }}
   {{- end }}
-{{- if eq $.Values.global.options.syncPolicy "Automatic" }}
+  {{- if .syncPolicy }}
+  syncPolicy: {{ .syncPolicy | toPrettyJson }}
+  {{- else }}
```

Comment:
What about `{{ else if eq $.Values.global.options.syncPolicy "Automatic" }}` ?

Reply:
a) We don't do that anywhere else
b) If that variable _isn't_ set, the resulting config might be invalid/inconsistent in weird ways. This would remove the inconcistency

Reply:
This also violates one of my personal pet peeves, of having two different variables named the same thing but having different types (string vs. object). I think the global variable needs to go.